erpnext/patches/v13_0/set_status_in_maintenance_schedule_table.py

```diff
@@ -0,0 +1,8 @@
+import frappe
+
+def execute():
+	frappe.db.sql("""
+		UPDATE `tabMaintenance Schedule Detail`
+		SET completion_status = 'Pending'
+		WHERE docstatus = 1
```

Comment:
Include legacy draft schedules too (that were created just before this refactor was available)
```suggestion
		WHERE docstatus < 2
```